Mathlib/Order/KrullDimension.lean

```diff
@@ -162,6 +164,36 @@ lemma height_mono : Monotone (α := α) height :=
 @[gcongr] protected lemma _root_.GCongr.height_le_height (a b : α) (hab : a ≤ b) :
     height a ≤ height b := height_mono hab
 
+private lemma height_add_const (a : α) (n : ℕ∞) :
```

Comment:
Review comment: This helper lemma is used more often in https://github.com/leanprover-community/mathlib4/pull/15524, hence extracted.